subsys/mgmt/mcumgr/grp/fs_mgmt/Kconfig

```diff
@@ -125,8 +125,9 @@ config MCUMGR_GRP_FS_CHECKSUM_IEEE_CRC32
 
 config MCUMGR_GRP_FS_HASH_SHA256
 	bool "SHA256 hash support"
-	depends on BUILD_WITH_TFM || MBEDTLS_SHA256
-	select PSA_WANT_ALG_SHA_256 if BUILD_WITH_TFM
+	select PSA_WANT_ALG_SHA_256
+	select MBEDTLS if !BUILD_WITH_TFM
+	select MBEDTLS_PSA_CRYPTO_C if !BUILD_WITH_TFM
```

Comment:
doesn't PSA work without TFM?

Reply:
Yes, indeed here I'm enabling Mbed TLS' PSA Crypto implementation in case TF-M is not enabled.
In case I missed the point in your question, can you please try to rephrase it?